business_requirement_deliverable_cost/models/business.py

```diff
@@ -102,7 +103,7 @@ def product_id_change(self):
                     pricelist=pricelist_id.id,
                     uom=self.uom_id.id,
                 )
-                sale_price_unit = product.list_price
+                sale_price_unit = product.price
```

Comment:
`list_price` is a base price set by user and `price` is the computed field `Product price`. When context changes is re-calculated. Weird to the user check that if change `qty` or `uom_id` parameter will back to the base price without take on count the appropriate pricelist or unit of measure.